### DISCUSSION TITLE
move data from one server to another,how should i choose?

### DISCUSSION THREAD

**Discussion by zhujinghuic** [minio/minio#14374] [Q&A] [answered]
i found i can move data with scp tool，so what advantage using mc tool or using rclone tool? what is the difference between their?

**Comment by harshavardhana** [comment, marked as answer]
mc mirror

**Reply by zhujinghuic** [reply]
if i deploy only one node,whether the object resource has none json file? if i deploy as distribution，the object resource will split several shard files distribute in several nodes?

**Reply by klauspost** [reply]
Just read the docs, it is all in there: https://docs.min.io/minio/baremetal/installation/deployment-and-management.html

Single drive mode has json files for metadata, placed in `.minio.sys`.
